src/components/Composer.vue

```diff
@@ -348,6 +357,12 @@ export default {
 		onInputChanged() {
 			this.saveDraftDebounced(this.getMessageData())
 		},
+		onAddLocalAttachment() {
+			this.$refs.composerAttachments.onAddLocalAttachment()
```

Comment:
In general not a big fan of this coding style. This should be doable with events. Check https://stackoverflow.com/a/42634865/2239067 and https://stackoverflow.com/questions/43256978/event-from-parent-to-child-component/43258428#43258428. The former is less code, but requires a parent component (problematic for isolated testing), the latter requires a bit more code but is quite elegant and works with testing :)

Reply:
oh, I didn't even know that this way of doing it existed...
maybe it would even make sense to handle this in a vuex store? :thinking: 

anyway I'm happy to implement it the way you want :)

Reply:
so I implemented the second variant with the local event bus :)
hope this is fine now...

Reply:
I'd only use Vuex for global state. These attachments are local state, only really relevant to these components.